openedx/features/content_tagging/fixtures/system_defined.yaml

```diff
@@ -0,0 +1,33 @@
+- model: oel_tagging.taxonomy
+  pk: -1
+  fields:
+    name: Languages
+    description: ISO 639-1 Languages. Allows tags for any language configured for use on the instance
+    enabled: true
+    required: true
+    allow_multiple: false
+    allow_free_text: false
+    visible_to_authors: true
+    _taxonomy_class: openedx.features.content_tagging.models.ContentLanguageTaxonomy
+- model: oel_tagging.taxonomy
+  pk: -2
+  fields:
+    name: Organizations
+    description: Allows tags for any organization ID created on the instance.
```

Comment:
Actually remind me why we're making an Organization taxonomy? The org of every object is already encoded in its usage ID, so I thought we didn't need to use tags for organizations.

Reply:
These object tags will be pulled into the search engine for filtering by Org.
There's probably other ways to do this besides storing Org tags, but it seemed cleanest to use the tagging system instead of special casing this data.

Reply:
OK, makes sense. Would be nice to have that context in the code/docs, but maybe I just missed it.

Reply:
Comment added with this context [here](https://github.com/openedx/edx-platform/pull/32869/commits/8f66534dea0616c0f24150cadee4a22da4a80d0b#diff-1f01045e9b5f4e8b46c76dec746b3a595ff3b11513c3a538175c406ec98719c0R44)

Reply:
In other words, as we found out in our meeting yesterday, we already need "special casing" to handle indexing the text content of the XBlocks (which come from `block.index_dictionary()` and which are definitely not tags), so if we're already indexing text content as a separate field unrelated to tags, I think it's probably going to be easier to index the org in the same way as the keywords, and pull it from the usage key, rather than doing all this work to build an org taxonomy, keep it in sync with the org models, and automatically tag every XBlock. But we can go either way, and no need to change anything.